apps/docs/docs/tutorials/api-sdk/onchain-invites.md

```diff
@@ -0,0 +1,172 @@
+---
+sidebar_position: 2
+title: On-chain invites
+---
+
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+
+# Using invite codes with on-chain groups
+
+This tutorial will guide you through the complete process of adding a user to an on-chain group via invite codes.
+
+## Install library
+
+<Tabs
+defaultValue="npm"
+groupId="package-managers"
+values={[
+{label: 'npm', value: 'npm'},
+{label: 'Yarn', value: 'yarn'},
+{label: 'pnpm', value: 'pnpm'}
+]}>
+<TabItem value="npm">
+
+```bash
+npm install @bandada/utils
+```
+
+</TabItem>
+<TabItem value="yarn">
+
+```bash
+yarn add @bandada/utils
+```
+
+</TabItem>
+<TabItem value="pnpm">
+
+```bash
+pnpm add @bandada/utils
+```
+
+</TabItem>
+</Tabs>
+
+## Create a new instance
+
+After installing the utils package, create a new instance of `Semaphore`. 
+
+More details on the utils package can be found [here](https://github.com/bandada-infra/bandada/tree/main/libs/utils).
+
+- Create a new instance of Semaphore contract.
+
+```ts
+import { getSemaphoreContract } from "@bandada/utils"
+
+const semaphore = getSemaphoreContract("sepolia")
+```
+
+- Create a new instance of Semaphore contract with signer.
+
+```ts
+import { getSemaphoreContract } from "@bandada/utils"
+import { useSigner } from "wagmi"
+
+const { data: signer } = useSigner()
+
+const semaphore = getSemaphoreContract("sepolia", signer)
+```
+
+You can choose to use other signer package of your choice, wagmi is just an example.
+
+
+## Create a group
+
+Create a new on-chain group with associated group.
+
+```ts
+const semaphore = getSemaphoreContract("sepolia", signer as any)
+const admin = await signer.getAddress()
+
+// create on-chain group
+const receipt = await semaphore.createGroup(admin)
+
+const groupIdBigNumber = receipt.events?.[0]?.args?.[0].toString()
+const onchainGroupId = groupIdBigNumber.toString()
+
+const apiKey = "your-api-key"
+const groupCreationDetails = {
+    name: onchainGroupId,
+    description: `This group is associated to the on-chain group ${onchainGroupId}`,
+    treeDepth: 16,
+    fingerprintDuration: 3600
+}
+
+// create associated group
+const group = apiSdk.createGroup(groupCreationDetails, apiKey)
+```
+
+## Add members to on-chain group using invite code
+
+Create a new group invite.
+
+```ts
+const groupId = "your-associated-group-id"
+const apiKey = "your-api-key"
+
+const invite = await apiSdk.createInvite(groupId, apiKey)
+```
+
+## Add member using an invite code
+
+Adds a member to an on-chain group using an invite code.
+
+```ts
+const groupId = "your-group-id"
+const memberId = "member-id"
+const inviteCode = "INVITECODE"
+
+const group = await semaphore.getGroup(groupId, {
+    members: true
+})
+
+const invite = await apiSdk.getInvite(inviteCode)
+
```

Comment:
Before adding the member, the invite should be checked with the [`getInvite`](https://docs.bandada.pse.dev/api-sdk#get-invite) function. If `redeemed` is false, the member can be added, if it's true, the member can't be added because that invite was already used.

(The same in the full example)